SP5WWP/m17-packet/m17-packet-encode.c

```diff
@@ -98,22 +98,22 @@ void fill_data(float* out, uint16_t* cnt, const uint8_t* in)
 }
 
 //convert a user string (as hex octets) into a uint8_t array for raw packet encoding
-void parse_raw_user_string (char * input)
+void parse_raw_user_string(char* input)
```

Comment:
Can we replace this function with [the one I wrote](https://github.com/M17-Project/M17_Implementations/blob/5f7c1df97b7513b39382eafcfc91b1b7b54119d0/SP5WWP/m17-coder/m17-coder-sym.c#L154)?